insights/parsers/dig.py

```diff
@@ -0,0 +1,171 @@
+"""
+Domain information groper (Dig) parsers
+=======================================
+
+Parsers included in this module are:
+
+DigDnssec - command ``/usr/bin/dig +dnssec . SOA``
+--------------------------------------------------
+DigEdns - command ``/usr/bin/dig +edns=0 . SOA``
+------------------------------------------------
+DigNoedns - command ``/usr/bin/dig +noedns . SOA``
+--------------------------------------------------
+"""
+
+import re
+
+from insights.core import Parser
+from insights.core.plugins import parser
+from insights.parsers import SkipException
+from insights.specs import Specs
+
+
+HEADER_TEMPLATE = re.compile(r';; ->>HEADER<<-.*status: (\S+),')
+RRSIG_TEMPLATE = re.compile(r'RRSIG')
+
+
+class Dig(Parser):
```

Comment:
@jobselko this should use the `CommandParser` base class instead to check for some of the typical error messages commands return.